modules/ROOT/pages/logging/log-downloads.adoc

```diff
@@ -11,43 +11,48 @@
 label:AuraDB-Professional[]
 label:AuraDB-Virtual-Dedicated-Cloud[]
 
-Both *Query log analyzer* and *Security log analyzer* allow you to download the fetched logs.
-The downloaded logs account for any filters and search that have been applied.
+Downloading logs requires a project role of either _{role-project-admin}_ or _{role-project-member}_.
+
+You can download logs from the *Query log* and the *Security log*.
+The logs include any filters or search terms applied when the logs were fetched.
 Downloading logs consists of two steps: **initiate the download** and **download the file**.
 
-[NOTE]
-====
-Downloading logs requires a project role of either _{role-project-admin}_ or _{role-project-member}_.
-====
+== Initiate logs 
 
-== Initiate download
+Each request is up to {max-request-hours} hours of data and a maximum of {max-download-rows} rows.
 
-To initiate a download, after fetching the logs, use the _Initiate log download_ icon button (Arrow Down Tray Icon) in the top right corner of either the Summary or Details table.
-Select the following options in the dialog:
+After fetching logs, use the *Initiate logs download* button (arrow down tray icon) in the top right corner of the *Summary* and *Details* table. 
 
-* *Log type* - Select one or both:
-** *Summary* - Aggregated log statistics
-** *Details* - Individual log events
-* *Format* - Choose your preferred format:
-** *JSON*
-** *CSV* with optional settings:
-*** Include CSV headers (default: enabled)
-*** CSV field delimiter (default: comma)
+* Select the *log type*: Summary (aggregated log statistics), Details (individual log events), or both. 
+* Choose a format: JSON or CSV. 
+If you choose CSV, you can optionally include CSV headers (enabled by default) and specify a field delimiter (default: comma).
+* Select *Confirm* to request the logs.
 
-*Confirm* starts the download.
+When the requested log is ready, it appears in the *Downloads archive.*
+
+== Download log
+
+Download logs from the *Downloads archive*.
+When the status is Ready, select the Download icon to download the file.
+
+Logs are provided as a zipped file in your chosen format (JSON or CSV), containing the same fields shown in the Field Name columns of the Summary and Details tables above.
+The file name follows this pattern: `<db_id>-<timestamp>-<log_type>-logs-<log_sub_type>.<format>.gz`, for example: `dd9ba752-1731586207476-query-logs-details.json.gz`
+
+To delete a log file, select the delete icon in the Actions column.
 
 [NOTE]
 ====
-Each download request is limited to {max-request-hours} hours of data and a maximum of {max-download-rows} rows.
+Log files are automatically deleted from the downloads archive after {log-download-retention-days} days.
 ====
 
-== Download the file
+== Downloads archive
+
+To access the archive click the *Open downloads archive* icon (Document Text icon) in the top-right corner of the page.
```

Comment:
```suggestion
To access the archive click the *Open downloads archive* icon in the top-right corner of the page.
```
Instead of trying to describe it, maybe we should have s screenshot?